### DISCUSSION TITLE
国内镜像网站被锁定导致Alas无法更新

### DISCUSSION THREAD

**Discussion by LouiseShariarc** [LmeSzinc/AzurLaneAutoScript#2692] [Q&A]
报错如下
`+---------------------------------------------------+
|                    UPDATE ALAS                    |
+---------------------------------------------------+
==================== GIT INIT ====================
"D:/Program Files/AzurLaneAutoScript/toolkit/Git/mingw64/bin/git.exe" init
Reinitialized existing Git repository in D:/Program Files/AzurLaneAutoScript/.git/
[ success ]
==================== SET GIT PROXY ====================
"D:/Program Files/AzurLaneAutoScript/toolkit/Git/mingw64/bin/git.exe" config --local --unset http.proxy
[ allowed failure ], error_code: 5
"D:/Program Files/AzurLaneAutoScript/toolkit/Git/mingw64/bin/git.exe" config --local --unset https.proxy
[ allowed failure ], error_code: 5
"D:/Program Files/AzurLaneAutoScript/toolkit/Git/mingw64/bin/git.exe" config --local http.sslVerify true
[ success ]
==================== SET GIT REPOSITORY ====================
"D:/Program Files/AzurLaneAutoScript/toolkit/Git/mingw64/bin/git.exe" remote set-url origin https://e.coding.net/llop18870/alas/AzurLaneAutoScript.git
[ success ]
==================== FETCH REPOSITORY BRANCH ====================
"D:/Program Files/AzurLaneAutoScript/toolkit/Git/mingw64/bin/git.exe" fetch origin master
remote: CODING 提示: team 3532843 is locked
fatal: unable to access 'https://e.coding.net/llop18870/alas/AzurLaneAutoScript.git/': The requested URL returned error: 403
[ failure ], error_code: 128
+-----------------------------------------------------+
|                    UPDATE FAILED                    |
+-----------------------------------------------------+
==================== SHOW DEPLOY CONFIG ====================
Repository: https://e.coding.net/llop18870/alas/AzurLaneAutoScript.git
PypiMirror: https://pypi.tuna.tsinghua.edu.cn/simple
EnableRemoteAccess: True
SSHServer: app.hk1.azurlane.cloud:10022
SSHExecutable: C:/Windows/System32/OpenSSH/ssh.exe
Language: zh-CN
Theme: dark
Rest of the configs are the same as default

Last command: "D:/Program Files/AzurLaneAutoScript/toolkit/Git/mingw64/bin/git.exe" fetch origin master
Please check your deploy settings in config/deploy.yaml and re-open Alas.exe
Take the screenshot of entire window if you need help`

**Comment by illustriouslily** [comment]
+1

**Reply by timedosentlie** [reply]
+1

**Comment by asdyeyue** [comment]
+1

**Comment by Btzy1** [comment]
+1

**Comment by ChenLily1019** [comment]
+1

**Comment by gugugu1145114** [comment]
去AzurLaneAutoScript\config\deploy.yaml里面，把 Repository后面的地址改成https://github.com/LmeSzinc/AzurLaneAutoScript试试

**Reply by ChenLily1019** [reply]
可以了，需要科学一下

**Reply by LouiseShariarc** [reply]
就是因为没有科学所以才会用国内镜像啊（

**Reply by ChenLily1019** [reply]
主要是觉得开个脚本还要先挂V怪麻烦的，所以才用了国内的镜像

**Comment by manmanzhou** [comment]
+1 改成github的源可以更

**Comment by 36614609** [comment]
改成github的源还是不可用怎么办？
==================== FETCH REPOSITORY BRANCH ====================
"D:/Game/AzurLaneAutoScript/toolkit/Git/mingw64/bin/git.exe" fetch origin master
fatal: unable to access 'https://github.com/LmeSzinc/AzurLaneAutoScript/': SSL certificate problem: unable to get local issuer certificate
[ failure ], error_code: 128
+-----------------------------------------------------+
|                    UPDATE FAILED                    |
+-----------------------------------------------------+
==================== SHOW DEPLOY CONFIG ====================
PypiMirror: https://pypi.tuna.tsinghua.edu.cn/simple
Language: zh-CN
Theme: dark
Rest of the configs are the same as default

Last command: "D:/Game/AzurLaneAutoScript/toolkit/Git/mingw64/bin/git.exe" fetch origin master
Please check your deploy settings in config/deploy.yaml and re-open Alas.exe
Take the screenshot of entire window if you need help

**Comment by LmeSzinc** [comment]
https://github.com/LmeSzinc/AzurLaneAutoScript/issues/2687#issuecomment-1572526474
